src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -403,8 +419,8 @@ bool CHyprGroupBarDecoration::onMouseButtonOnDeco(const Vector2D& pos, wlr_point
     if (pWindow != m_pWindow)
         pWindow->setGroupCurrent(pWindow);
 
-    if (!g_pCompositor->isWindowActive(pWindow))
-        g_pCompositor->focusWindow(pWindow);
```

Comment:
why was this removed?

Reply:
Seemed unnecessary, since `setGroupCurrent` already called `focusWindow`. I probably removed it because it worked the same way without it while I was testing. But I don't really remember.